<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comm Audio Server</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="srvServerGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Server</string>
    </property>
    <widget class="QPushButton" name="srvStartStopButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="srvStatusLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status: </string>
     </property>
    </widget>
    <widget class="QLabel" name="srvTCPPortLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP Port</string>
     </property>
    </widget>
    <widget class="QLabel" name="srvUDPPortLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP Port</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="srvTCPPortSpinner">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>7000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="srvUDPPortSpinner">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>7080</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="srvClientListGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>211</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Client List</string>
    </property>
    <widget class="QListWidget" name="srvClientListWidgest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>481</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Track</string>
    </property>
    <widget class="QPushButton" name="srvTrackPlayPauseButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="srvTrackFFButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="srvTrackNextButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="srvTrackRWButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="srvTrackPreviousButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="musicList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>421</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Track List</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="srvShuffleRadioButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Shuffle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="srvSelectPlaylistButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Playlist</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
